### PR TITLE
Format the package manifest

### PR DESCRIPTION
**Pull request by ekohl**
This was automatically formatted using [obal-fmt](https://github.com/theforeman/obal/pull/59).

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -269,25 +265,25 @@ katello_packages:
...
   vars:
-    package_base_dir: 'packages/katello/'
+    package_base_dir: packages/katello/
     releasers:
-    - koji-katello-client
-  hosts:
-    katello-host-tools: {}
-    katello-repos: {}
+    - koji-katello
+packages:
```
Comment (ehelms) [inline package_manifest.yaml]: I kinda liked this being at the top as it gave a general overview of the groups I expect to find below.

Comment (ehelms) [reply package_manifest.yaml]: Perhaps this can be a convention that is always at the top?

File: package_manifest.yaml [hunk 2]
```diff
@@ -140,115 +232,19 @@ foreman_packages:
...
-    rubygem-rsec: {}
-    rubygem-rubyipmi: {}
-katello_packages:
   vars:
-    package_base_dir: 'packages/katello/'
+    package_base_dir: packages/foreman/
     releasers:
-    - koji-katello
+    - koji-foreman
+katello_client_packages:
```
Comment (ehelms) [inline package_manifest.yaml]: Possible to have the formatter put empty lines between groups? Would be easier to spot start and stop of a group that way.

File: package_manifest.yaml (hunk 1 above)
Comment (ekohl) [reply package_manifest.yaml]: I like that too and will see how I can force the formatter to behave.

File: package_manifest.yaml (hunk 2 above)
Comment (ekohl) [reply package_manifest.yaml]: agreed

File: package_manifest.yaml [hunk 3]
```diff
@@ -1,20 +1,104 @@
----
-packages:
-  children:
-    foreman_packages: {}
-    plugins_packages: {}
-    katello_packages: {}
-    katello_client_packages: {}
-
 foreman_packages:
-  vars:
```
Comment (ekohl) [inline package_manifest.yaml]: I also liked vars above hosts because it's usually shorter and sort of a preset for the rest.

**Comment by ekohl** [comment]
Closing this for now.

**Pull request closed without merging**